features/list_support_actions.feature

```diff
@@ -46,4 +46,20 @@ Feature: List support actions
   Scenario: Sort support actions by category descending
     Given a resident with "Book drops and entertainment, Dog walking, Groceries and cooked meals" support actions
     When I sort support actions by category in "DESC" order
-    Then I see the support action for category "Groceries and cooked meals" first in the results
\ No newline at end of file
+    Then I see the support action for category "Groceries and cooked meals" first in the results
+
+  Scenario: See unassigned needs I have created in the created and unassigned list
+    Given a unique resident
+    And I add support actions "Dog walking"
+    And I submit the add support actions form
+    When I go the the created and unassigned support action list
+    Then I see the support action for category "Dog walking" in the results
+
+  @javascript
+  Scenario: Do not see assigned needs I have created in the created and unassigned list
+    Given a unique resident

```

Comment:
You could compress this into "Given I have created a support action "Dog Walking""
Which calls
```
step  a unique resident
step I add support actions "Dog walking"
step I submit the add support actions form
step I assign the support action to another user
```

It does not make any difference expect for it reading a bit easier

Reply:
Fair point, have updated